Argo/Globals/JSONValue.swift

```diff
@@ -18,8 +18,8 @@ public extension JSONValue {
 
     case let v as [String: AnyObject]:
       let object = reduce(v.keys, JSONDict()) { accum, key in
-        let append = curry(Dictionary.appendKey)(accum)(key)
-        return (append <^> v[key] >>- self.parse) ?? append(.JSONNull)
+        let value = self.parse(v[key]!)
```

Comment:
We should never explicitly unwrap anything for safety reasons. If the bind (`>>-`) is causing bad access then we should try and replace it with an equivalent that does work.